cves/kernel/CVE-2016-4578.yml

```diff
@@ -364,8 +382,11 @@ forgotten_check:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: false
+  note: |
```

Comment:
Replace your semicolon with a period. Remove the word "always" from your second sentence. Add punctuation.